CAIPs/caip-169.md

```diff
@@ -0,0 +1,516 @@
+defining their proof formats are specified in the [VC spec][] itself.
+
+#### Embedded Proofs
+
+CPs MUST support the following LD-Proof types for embedded proofs (i.e. VC-LDP):
+- [`EthereumEip712Signature2021`](https://w3id.org/security/suites/eip712sig-2021)
+- [`JsonWebSignature2020`](https://w3id.org/security/suites/jws-2020), only
+  Ed25519 and secp256k1
+
+CPs are RECOMMENDED to support the following LD-Proof types for wide support of
+VC formats with embedded proofs (i.e. VC-LDP):
+- [`BbsBlsSignature2020`](https://w3id.org/security/suites/bls12381-2020)
+- [`BbsBlsBoundSignature2020`](https://w3id.org/security/suites/bls12381-2020)
+
+#### External Proofs
+
+CPs SHOULD support the following
+[IANA](https://www.iana.org/assignments/jose/jose.xhtml) JWS algorithms for
+external proofs (i.e. VC-JWT):
+- [`ES256K`](https://www.rfc-editor.org/rfc/rfc8812.html)
+- [`EdDSA`](https://www.rfc-editor.org/rfc/rfc8037.html)
+
+A helpful test suite for conformance-testing VC-JWT signing can be found at [JWS-test-suite][].
+
+### Supported Verifiable Credentials Profile
+
+VCs that can be used with this specification MUST be valid JSON-LD as defined in
+the [VC spec][]. VCs and VPs MUST use either the proofs recommended by [Embedded
+Proofs](#EmbeddedProofs) or [External Proofs](#ExternalProofs) above to conform
+to this specification; implementations are RECOMMENDED to limit themselves to
+algorithms and key types corresponding to registered claim formats in the [DIF
+Claim Format registry][] if they are expecting reasonable interoperability in
+the near term, and to carefully construct their [presentation_definition
+object][] accordingly.
+
+### Store
+
+Stores the given VC in the CP.
+
+#### Method:
+
+`wallet_creds_store`
+
+##### Params:
+
+- `vc` - A Verifiable Credential.
+
+##### Returns:
+
+- `error` - OPTIONAL. If `vc` was malformed or does not comply with the
+  Verifiable Credentials Profile defined in this specification. Note that some
+  wallets will call `wallet_creds_verify` locally or remotely and pass back an
+  error message received thereby, depending on security context.
+
+### Verify
+
+Verifies the proof section of a single verifiable credential after dereferencing
+its `issuer` property for key material. Note that in an application<>wallet
+connection, the application calls the wallet for the wallet to either perform
+verification locally or remotely; in either case, the application will await
+asynchronously for a success or error code, regardless of how the wallet
+verifies the passed credential. Wallets that cannot return appropriate error
+codes back MUST NOT authorize apps to call this method as undefined behavior may
+occur.
+
+#### Method:
+
+`wallet_creds_verify`
+
+##### Params:
+
+- `vc` - A Verifiable Credential.
+
+##### Returns:
+
+- `error` - OPTIONAL. If `vc` was malformed or does not comply with the
+  Verifiable Credentials Profile defined in this specification.
+
```

Comment:
Does this need to add any detail about expected implementation/behavior of VC verification?